src/test/unit/types/SdrTest.cpp

```diff
@@ -151,15 +151,23 @@ TEST(SdrTest, TestReshape) {
 
 TEST(SdrTest, TestSetDenseVec) {
     SDR a({11, 10, 4});
-    Byte *before = a.getDense().data();
-    SDR_dense_t vec = vector<Byte>(440, 1);
-    Byte *data = vec.data();
+    auto before = a.getDense().data();
+    SDR_dense_t vec = SDR_dense_t(440, 1);
+    auto data = vec.data();
     a.setDense( vec );
-    Byte *after = a.getDense().data();
+    auto after = a.getDense().data();
     ASSERT_NE( before, after ); // not a copy.
     ASSERT_EQ( after, data ); // correct data buffer.
 }
 
+TEST(SdrTest, TestSetDenseUInt) {
+    SDR a({11, 10, 4});
+    auto vec = SDR_dense_t(a.size, 1);
```

Comment:
This data type should not have changed.  The test name explicitly calls for `UInt` not `Byte`

Reply:
> The SDR has a big matrix of different data types which should work, even though they are not the exact same type which the SDR uses internally. 

yes, I made a misunderstood these. will need to get all of those back.

Reply:
> Also, I don't see any changes to VectorHelpers. Isn't the purpose of this PR to remove it? 

yes, I've just updated the old PR to see how it fares. VectorH should be removed in this PR. 

Also, I'm getting failing "exact output" tests but I don't see anything that should cause that (at the first glance)